packages/jest-runtime/src/index.js

```diff
@@ -361,14 +353,7 @@ class Runtime {
   }
 
   getAllCoverageInfo() {
-    const coverage = Object.create(null);
-    if (this._config.collectCoverage) {
-      const collectors = this._coverageCollectors;
-      for (const filePath in collectors) {
-        coverage[filePath] = collectors[filePath].extractRuntimeCoverageInfo();
-      }
-    }
-    return coverage;
+    return this._environment.global.__coverage__;
```

Comment:
whatever sets the flag, it would be nice if it did `Object.defineProperty` so that it isn't enumerable.